app/mailers/mailer.rb

```diff
@@ -1,11 +1,8 @@
-class CommunicartMailer < ApplicationMailer
+class Mailer < ApplicationMailer
```

Comment:
It is generic. Right now it is really just the base mailer for this project. In rails apps, I usually name that `Mailer`. 

I agree that we should move more methods out of this and into smaller mail classes but was trying to keep this PR under 400 changes per my usual rule of thumb. 

What do you think about keeping this name for now as the base mailer? 

I am not sure that a name like `ProposalActivityMailer` would improve the clarity of purpose of this class given the generic nature of `Activity` -- would rather stick with a generic name and then create more specific classes like we have with `CancellationMailer` and `ReportMailer`